src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -541,7 +541,7 @@ Map<String, Object> setEvidenceReceivedNotificationData(Map<String, Object> pers
     }
 
     private Map<String, Object> setAppealReceivedDetails(Map<String, Object> personalisation, EventDetails eventDetails, SscsCaseData ccdResponse) {
-        LocalDate localDate = eventDetails.getDateTime().plusDays(MAX_DWP_RESPONSE_DAYS).toLocalDate();
+        LocalDate localDate = eventDetails.getDateTime().plusDays(dwpResponseUtil.calculateMaxDwpResponseDays(ccdResponse.getBenefitCode())).toLocalDate();
```

Comment:
if calculateMaxDwpResponseDays is not used anywhere else, you can just add it in this class